ethportal-peertest/src/events.rs

```diff
@@ -49,14 +46,26 @@ impl PortalnetEvents {
             match std::str::from_utf8(request.protocol()) {
                 Ok(protocol) => match protocol {
```

Comment:
FYI this match will not work with the latest ProtocolId update in https://github.com/ethereum/trin/pull/162. I'll create an issue to fix this in a separate PR.